### PR TITLE
libetpan 1.7.2 (new formula)

### PR DESCRIPTION
**Pull request by yurikoles**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
This formulae is needed for claws-mail.

### CODE REVIEW

File: Formula/libetpan.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
```
Comment (fxcoudert) [inline Formula/libetpan.rb]: Remove empty line 1

File: Formula/libetpan.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+class Libetpan < Formula
+  desc "Mail library is to provide a portable, efficient framework."
```
Comment (fxcoudert) [inline Formula/libetpan.rb]: No period at the end of `desc`.
The sentence should be reworked, maybe _“Portable mail library handling several protocols”_?

File: Formula/libetpan.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  def install
+    system "./autogen.sh", "--disable-debug",
+                           "--disable-dependency-tracking",
+                           "--disable-silent-rules",
+                           "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/libetpan-config", "--version"
```
Comment (fxcoudert) [inline Formula/libetpan.rb]: The standard test for C libraries is to compile a simple source code and link it against the library.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/libetpan.rb (hunk 3 above)
Comment (yurikoles) [inline Formula/libetpan.rb]: There is a test directory inside source tree. I'm not sure how to use it.

**Comment by yurikoles** [comment]
Is there a way how to use test dir for test of this keg?

File: Formula/libetpan.rb (hunk 3 above)
Comment (fxcoudert) [inline Formula/libetpan.rb]: Don't retain the test directory, just include a short example (up to 10 lines) in C inside the `test` block of the formula. Look, for example, at how `zita-convolver.rb` does this.

**Comment by yurikoles** [comment]
@fxcoudert I'm not sure how to do it, since most tests require a real email account and server setup.

**Comment by yurikoles** [comment]
@MikeMcQuaid please merge.

**Comment by fxcoudert** [comment]
We will need a test for this. If we cannot have a run test, they at least compile some C code and link it with the library to check that things are alright. Possibly print the version number from this C code and check it.

**Comment by yurikoles** [comment]
@fxcoudert done

**Comment by ilovezfs** [comment]
No release since May of last year. Is this actually maintained?

**Comment by yurikoles** [comment]
@ilovezfs there was commits a week ago https://github.com/dinhviethoa/libetpan/commits/master

Nevertheless this formula is required by claws-mail which I plan to introduce after merge of this formulae.

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @yurikoles for your contribution to Homebrew!

**Comment by yurikoles** [comment]
@fxcoudert you welcome.
But why test doesn't print version to stdout?

**Comment by bfontaine** [comment]
@yurikoles It would if you used `brew test --verbose libetpan` :wink:

**Pull request closed without merging**